app/views/insured/families/_enrollment_refactored.html.erb

```diff
@@ -122,6 +129,14 @@
         </div>
       </div>
     </div>
+
+    <script>
+      function handleContactInfoKeyDown(event, id, modalId) {
+        if (event.key === 'Enter' || event.key === ' ') {
+          document.getElementById(id).click();
+        }
+      }
+    </script>
```

Comment:
small nit, could we move this function to https://github.com/ideacrew/enroll/blob/clean-bootstrap-4-upgrade/app/assets/javascripts/keyboard_navigation.js#L1 potentially?